pubspec.yaml

```diff
@@ -1,63 +1,66 @@
 name: embla
 description: Icelandic-language voice assistant.
-publish_to: 'none'
+publish_to: "none"
 version: 1.3.1+15
 
 environment:
-  sdk: ">=2.10.0 <3.0.0"
+    sdk: ">=2.10.0 <3.0.0"
 
 dependencies:
```

Comment:
Isn't YAML usually formatted with two-space indentation? That's what VS Code does by default.